src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -65,35 +54,54 @@ const createReleaseServiceInstalls = async (
 	}
 	const serviceIds = services.map(({ id }) => id);
 
-	const serviceInstalls = (await api.get({
-		resource: 'service_install',
+	const missingServiceFilters = serviceIds.map((serviceId) => ({
+		$not: {
+			service_install: {
+				$any: {
+					$alias: 'si',
+					$expr: {
+						si: {
+							installs__service: serviceId,
+						},
+					},
+				},
+			},
+		},
+	}));
```

Comment:
The tests seem to be able to run the query even when using 200 services ids (in which case the ORs and the permissions make it really big).